lib/index.js

```diff
@@ -44,6 +44,14 @@ module.exports = function (gulp, options) {
     mocha: {}
   });
 
+  // allows command line arguments to modify the options object
+  // ex: gulp test --mocha.grep @production-only --mocha.invert
+  // -- options.mocha == { grep: '@production-only', invert: true }
+  _.each(argv, (value, name) => {
+    if (name === '_' || name === '$0') return;
+    options[name] = _.merge(options[name] || {}, value);
```

Comment:
Never mind, looks like you're parsing the command line with yargs, so that's where the magic happens.